src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -173,4 +184,40 @@ private static boolean match(String one, String two) {
     two = two.toLowerCase().trim();
     return one.contains(two) || two.contains(one);
   }
-}
+
+  /**
+   * Returns product name from {@link CpeMatch#getCpe23Uri()}.
+   */
+  private String productFrom23Uri(String cpe23Uri) {
+    return cpe23Uri.split(":")[4];
+  }
+
+  /**
+   * Returns product name from {@link CpeMatch#getCpe22Uri()}.
+   */
+  private String productFrom22Uri(String cpe22Uri) {
+    return cpe22Uri.split(":")[3];
+  }
+
+  /**
+   * Checks if the term1 is similar to term2.
+   * 
+   * @param term1 to be compared with term2.
+   * @param term2 to be compared with term1.
+   * @return True if the accumulated points is greater than threshold. Otherwise False.
+   */
+  private boolean metric(String term1, String term2) {
+    int point = 0;
+
+    if ((new JaroWinklerDistance()).apply(term1, term2) > METRIC_THRESHOLD) {
+      point += 1;
+    }
+
+    if ((new LongestCommonSubsequence()).apply(term1,
+        term2) >= (term1.length() * METRIC_THRESHOLD)) {
```

Comment:
Please don't break the line here in the list of parameters. Please try to simplify this condition (for example, you can define methods, variable with meaningful names).